src/ray/protobuf/common.proto

```diff
@@ -422,6 +423,13 @@ message TaskSpec {
   // This will be the actor creation task's task id for concurrent actors. Or
   // the main thread's task id for other cases.
   bytes submitter_task_id = 33;
+  // Some timestamps of the task's lifetime, useful for metrics.
+  // The time the task's dependencies have been resolved.
+  google.protobuf.Timestamp dependency_resolution_time = 34;
```

Comment:
Not sure if there's a reason we haven't been following the best practices around well known types? 

https://protobuf.dev/programming-guides/dos-donts/#do-use-well-known-types-and-common-types

Reply:
because the generated java code has a dependency conflict